File: Domain/Entities/ET/Evaluation.cs
using System;
using System.Collections.Generic;

namespace Domain.Entities.ET;

public class Evaluation : EntityBase
{
    public Guid Id { get; set; }
    public string EmployeeCode { get; set; }
    public DateTime? StartDate { get; set; }
    public DateTime? EndDate { get; set; }
    public Guid Status { get; set; }
    public string SoftSkills { get; set; }
    public string HardSkills { get; set; }
    public string Goals { get; set; }
    public ICollection<EvaluationDetail> EvaluationDetails { get; set; } = new List<EvaluationDetail>();
    public ICollection<SkillMatrix> SkillMatrices { get; set; } = new List<SkillMatrix>();
}

File: Web/ClientApp/src/app/models/et/evaluation.ts
import { Guid } from "guid-typescript";
import { EntityBase } from "../entityBase";
import { EvaluationDetail } from "./evaluationDetail";
import { SkillMatrix } from "./skillMatrix";

export class Evaluation extends EntityBase {
    id: Guid;
    employeeCode: string;
    startDate?: Date;
    endDate?: Date;
    status?: Guid;
    skillMatrixId: Guid;
    skillMatrixMasterId: Guid;
    evaluationDetails: EvaluationDetail[] = []
    skillMatrices: SkillMatrix[] = []
}